macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -383,8 +383,6 @@ final class AddressBarButtonsViewController: NSViewController {
     }
 
     @IBAction func aiChatButtonAction(_ sender: Any) {
-        PixelKit.fire(AIChatPixel.aiChatAddressBarButtonClicked, frequency: .dailyAndCount, includeAppVersionParameter: true)
```

Comment:
This is moved down to both `func toggleAIChatSidebar(for tab: Tab)` and `func openAIChatTab(for tab: Tab, with behavior: LinkOpenBehavior)` that fire this pixel with different params.